client/java/src/main/java/io/openlineage/client/circuitBreaker/TaskQueueCircuitBreaker.java

```diff
@@ -29,15 +31,17 @@
  */
 @Slf4j
 public class TaskQueueCircuitBreaker implements CircuitBreaker {
+
+  public static final String TASK_QUEUE_METRIC_PREFIX = "openlineage.circuitBreaker.taskQueue.";
+  public static final String DROPPED_METRIC = TASK_QUEUE_METRIC_PREFIX + "dropped";
+  public static final String TIMED_OUT_METRIC = TASK_QUEUE_METRIC_PREFIX + "timedOut";
+  public static final String FAILED_METRIC = TASK_QUEUE_METRIC_PREFIX + "failed";
```

Comment:
I'm not that familiar with naming conventions in Java metrics. Shouldn't they be `hyphen-separated-lowercase`?

Reply:
Metrics collected by java client are: https://openlineage.io/docs/client/java/configuration#metrics-collected-by-java-client like `openlineage.circuitbreaker.engaged`. You're right they're not camel case, but they're not `hyphen-separated-lowercase` either. 

Option (1): make it lower case "openlineage.circuitbreaker.taskqueue.timedout"
Option (2): hyphen lowercase "openlineage.circuit-breaker.task-queue.timed-out"

Reply:
Let's keep it consistent with option 1 then, I guess.

Reply:
Option 1 please :)